README.md

```diff
@@ -33,7 +33,17 @@ Anna is part of KWOC : You can talk to us on Zulip or comment on issues to start
   - If I don't understand
   - Close tabs
   
-
+ # How to install ANNA
+ 
+1. Once you are on the repository page,look for the green “Clone or download” button on the right-hand side of the page.
+![imageedit_3_3326230850](https://user-images.githubusercontent.com/29461633/33730973-a91d3c7c-dba7-11e7-8efe-9d1b9f43980b.png)
```

Comment:
I think only this much part is required 
![deepinscreenshot_select-area_20171208003554](https://user-images.githubusercontent.com/12110844/33733459-da6e1e9c-dbaf-11e7-99f3-d2777dbe37bb.png)